lms/migrations/versions/37710e6bcb66_add_group_info_info_column.py

```diff
@@ -0,0 +1,30 @@
+"""
+Add group_info.info column.
+
+Revision ID: 37710e6bcb66
+Revises: a930adadac74
+Create Date: 2020-04-30 12:21:20.645402
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects.postgresql import JSONB
+from sqlalchemy.ext.mutable import MutableDict
+
+# revision identifiers, used by Alembic.
+revision = "37710e6bcb66"
+down_revision = "a930adadac74"
+
+
+def upgrade():
+    op.add_column(
+        "group_info", sa.Column("info", MutableDict.as_mutable(JSONB)),
+    )
+    group_info_table = sa.table(
+        "group_info", sa.Column("info", MutableDict.as_mutable(JSONB))
+    )
+    op.execute(group_info_table.update().values(info={"instructors": []}))
```

Comment:
It's normally a good idea to do schema and data migrations separately but I'm being lazy and combining these. There are only 2600 rows in the production `group_info` table so I think we're okay to update them all at once (it won't lock the DB for too long)